### DISCUSSION TITLE
Ultima template - what is purpose of css files for example layout.css

### DISCUSSION THREAD

**Discussion by kashcode** [primefaces/community#1339] [PrimeNG Templates] [answered]
I found that there is `layout.css` where should I use it, because there is also `layout.scss` if I modify `layout.scss` changes reflects on page, therefore I have question about `layout.css`

**Comment by melloware** [comment]
If have marked your topic correctly and this is for PrimeFaces and not for PrimeReact or NG etc then those SCSS are turned in CSS files by the SASS compiler.  So if you are using `sass:watch` every change you make instantly recompiles into the CSS file.

**Reply by kashcode** [reply]
Thanks, this is for Angular, will try build for production and check if it recompiles into css, because with `ng serve` css is not touched.

**Reply by melloware** [reply]
I moved your topic to PrimeNG Templates...

**Comment by cetincakiroglu** [comment, marked as answer]
Hi @kashcode ,

Probably it's a leftover from the update. You can safely delete it.
